controllers/operator/nmstate_controller.go

```diff
@@ -147,7 +147,8 @@ func (r *NMStateReconciler) applyManifests(instance *nmstatev1.NMState, ctx cont
 	}
 
 	isOpenShift, err := cluster.IsOpenShift(r.APIClient)
-	if err == nil && isOpenShift {
+	_, errUIPluginPathExists := os.Stat(filepath.Join("openshift", "ui-plugin"))
+	if err == nil && isOpenShift && errUIPluginPathExists == nil {
```

Comment:
just a small question: should we notify users via log that ui-plugin is not installed ?

Example:
```
if "NMState version supports ui-plugin" && errUIPluginPathExists != nil {
    r.Log.Info("NMState ui-plugin is not set, to enable this feature, please consult NMState documentation");
}
```

Reply:
i'm not sure, because if at this point the manifests of the ui plugin are missing from the operator container, the users can't do much in order to deploy the ui plugin at this point, since the operator is expecting the manifests to be included in the container.

Reply:
thanks! I will let others in the group review and approve :)